backend/eks/deploy.sh

```diff
@@ -21,14 +21,21 @@ pushd cap-terraform/eks || exit
 helm_init_client
 
 cat <<HEREDOC > terraform.tfvars
-region = "$EKS_LOCATION"
-workstation_cidr_block = "0.0.0.0/0"
-keypair_name = "$EKS_KEYPAIR"
-eks_version = "$EKS_VERS"
-cluster_labels = $EKS_CLUSTER_LABEL
-hosted_zone_id = "${EKS_ZONE_ID}"
-hosted_zone_name = "${EKS_ZONE_NAME}"
-hosted_zone_policy_arn = "${EKS_ZONE_POLICY}"
+cluster_name = "${EKS_CLUSTER_NAME}"
+region = "${EKS_LOCATION}"
+keypair_name = "${EKS_KEYPAIR}"
+eks_version = "${EKS_VERS}"
+cluster_labels = ${EKS_CLUSTER_LABEL}
+hosted_zone_name = "${EKS_HOSTED_ZONE_NAME}"
+external_dns_aws_access_key = "${AWS_ACCESS_KEY_ID}"
+external_dns_aws_secret_key = "${AWS_SECRET_ACCESS_KEY}"
+instance_type = "t2.large"
```

Comment:
This was removed by https://github.com/SUSE/catapult/pull/234. Do we need it back?